src/app/mini-games/custom-mini-game/custom-mini-game.component.ts

```diff
@@ -343,6 +343,7 @@ export class CustomMiniGameComponent implements OnInit, OnDestroy {
   onToggleSound(): void {
     this.soundService.toggleSound();
     this.isSoundOn = this.soundService.isSoundOn;
+    console.log('in onToggleSound. isSoundOn = ', this.isSoundOn);
```

Comment:
Do you want to monitor some issue using console.log?
Because it's better to remove such code before pushing